tests/polkadot_tests/contracts.rs

```diff
@@ -99,10 +99,7 @@ fn revert_constructor() {
     );
 
     runtime.constructor(0, Vec::new());
-
     runtime.function_expect_failure("test", Vec::new());
-
-    assert_eq!(runtime.output().len(), 0);
```

Comment:
why not assert that the "Hello, World!" revert has been sent?

Reply:
The contract doesn't return the output back to the caller. Should we bubble it up?

Reply:
I think we should. Also, now we can return `Result<T, LangError>` where `T` is what the function or constructor returns, and `LangError` is `(bytes4, string)`. I can do that in this PR or in a followup